enterprise/frontend/src/metabase-enterprise/data-studio/common/hooks/use-collection-path/useCollectionPath.ts

```diff
@@ -0,0 +1,26 @@
+import { skipToken } from "metabase/api/api";
+import { useGetCollectionQuery } from "metabase/api/collection";
+import { isRootCollection } from "metabase/collections/utils";
+import type { CollectionId } from "metabase-types/api/collection";
+
+interface UseCollectionPathOptions {
+  collectionId: CollectionId | null;
+}
+
+export const useCollectionPath = ({
+  collectionId,
+}: UseCollectionPathOptions) => {
+  const { data: collection, isLoading: isLoadingPath } = useGetCollectionQuery(
+    !collectionId ? skipToken : { id: collectionId },
+    { skip: !collectionId },
```

Comment:
This uses both approaches to skip a request at the same time, we can leave just one